### ISSUE TITLE
Target field auto-completion does not work

### ISSUE THREAD

**Issue by andreslotta** [johngodley/redirection#3591]
### Summary

Auto-completion when creating a redirect does not find / suggest existing posts from a custom post type.

### Actual Behavior

There is a post from a custom **event** post type with a title of **ŻfinMalta**. When I create a a redirect for a specific url (from the 404 list) and start typing something like "malta" or eben the identical post title into the target field the post is not found / suggested.

### Steps to reproduce

- Create a custom post type
- Create an instance of the custom post type
- Create a redirect and type parts of the post title into the target field
- See that the post not found / suggested

### Environment

- WP: 6.2
- Redirection: 5.3.9
- Browser: Chrome 111.0.5563.146 (Mac ARM) – Browser does not matter in this case though

Thanks for your help.

![CleanShot 2023-03-30 at 09 23 09@2x](https://user-images.githubusercontent.com/12016014/228760578-8a71fcd9-6aca-4ecd-bc28-8f85c369f5f1.png)

**Comment by johngodley** [comment]
Yes, it intentionally only shows posts as that is what the majority of people want.

**Comment by andreslotta** [comment]
> Yes, it intentionally only shows posts as that is what the majority of people want.

Thanks for the quick reply! Would it be an option to include a hook to enable additional post types if needed?

**Comment by Swelldev** [comment]
Is there an update on how to include custom post type pages to appear in the suggested list here?
